add -wd308 compiler option to avoid compiler warning #308 being treated as an error
Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
diff -Nur kohonen.orig/MD5 kohonen/MD5
--- kohonen.orig/MD5	2018-08-17 17:20:03.000000000 +0200
+++ kohonen/MD5	2019-04-05 16:19:20.927622744 +0200
@@ -65,3 +65,4 @@
 25ac1ecf2fd6446650632602f73ee35c *src/neighbourhood-functions.h
 a08c489f9b9a7fe83f5389456d2df1f6 *src/pbatch-supersom.cpp
 417171759d5123bf75830bd818230fa2 *src/supersom.cpp
+04d564ce60a31fc98e7a5b6159a78c3c *src/Makevars
diff -Nur kohonen.orig/src/Makevars kohonen/src/Makevars
--- kohonen.orig/src/Makevars	1970-01-01 01:00:00.000000000 +0100
+++ kohonen/src/Makevars	2019-04-05 16:18:01.699296172 +0200
@@ -0,0 +1,3 @@
+# disable Intel C++ compiler (icpc) warning/error #308, to avoid 'member "std::complex<double>::_M_value" is inaccessible'
+PKG_CFLAGS = -wd308
+PKG_CXXFLAGS = -wd308
